web/src/test/resources/snapshots/no.nav.modiapersonoversikt.consumer.infotrygd.consumer.foreldrepenger.mapping.ForeldrepengerMapperTest_dateMapping-0.json

```diff
@@ -1,153 +1 @@
-      "rangeDurationType" : null
-    } ]
-  } ], [ "org.joda.time.chrono.GJMonthOfYearDateTimeField", {
-    "durationField" : [ "org.joda.time.DurationField", {
-      "name" : "months",
-      "precise" : false,
-      "supported" : true,
-      "type" : [ "org.joda.time.DurationFieldType$StandardDurationFieldType", {
-        "name" : "months"
-      } ],
-      "unitMillis" : 2629746000
-    } ],
-    "leapDurationField" : [ "org.joda.time.field.PreciseDurationField", {
-      "name" : "days",
-      "precise" : true,
-      "supported" : true,
-      "type" : [ "org.joda.time.DurationFieldType$StandardDurationFieldType", {
-        "name" : "days"
-      } ],
-      "unitMillis" : 86400000
-    } ],
-    "lenient" : false,
-    "maximumValue" : 12,
-    "minimumValue" : 1,
-    "name" : "monthOfYear",
-    "rangeDurationField" : [ "org.joda.time.DurationField", {
-      "name" : "years",
-      "precise" : false,
-      "supported" : true,
-      "type" : [ "org.joda.time.DurationFieldType$StandardDurationFieldType", {
-        "name" : "years"
-      } ],
-      "unitMillis" : 31556952000
-    } ],
-    "supported" : true,
-    "type" : [ "org.joda.time.DateTimeFieldType$StandardDateTimeFieldType", {
-      "durationType" : [ "org.joda.time.DurationFieldType$StandardDurationFieldType", {
-        "name" : "months"
-      } ],
-      "name" : "monthOfYear",
-      "rangeDurationType" : [ "org.joda.time.DurationFieldType$StandardDurationFieldType", {
-        "name" : "years"
-      } ]
-    } ]
-  } ], [ "org.joda.time.chrono.BasicDayOfMonthDateTimeField", {
-    "durationField" : [ "org.joda.time.field.PreciseDurationField", {
-      "name" : "days",
-      "precise" : true,
-      "supported" : true,
-      "type" : [ "org.joda.time.DurationFieldType$StandardDurationFieldType", {
-        "name" : "days"
-      } ],
-      "unitMillis" : 86400000
-    } ],
-    "leapDurationField" : null,
-    "lenient" : false,
-    "maximumValue" : 31,
-    "minimumValue" : 1,
-    "name" : "dayOfMonth",
-    "rangeDurationField" : [ "org.joda.time.DurationField", {
-      "name" : "months",
-      "precise" : false,
-      "supported" : true,
-      "type" : [ "org.joda.time.DurationFieldType$StandardDurationFieldType", {
-        "name" : "months"
-      } ],
-      "unitMillis" : 2629746000
-    } ],
-    "supported" : true,
-    "type" : [ "org.joda.time.DateTimeFieldType$StandardDateTimeFieldType", {
-      "durationType" : [ "org.joda.time.DurationFieldType$StandardDurationFieldType", {
-        "name" : "days"
-      } ],
-      "name" : "dayOfMonth",
-      "rangeDurationType" : [ "org.joda.time.DurationFieldType$StandardDurationFieldType", {
-        "name" : "months"
-      } ]
-    } ],
-    "unitMillis" : 86400000
-  } ] ],
-  "monthOfYear" : 2,
-  "values" : [ 2013, 2, 13 ],
-  "weekOfWeekyear" : 7,
-  "weekyear" : 2013,
-  "year" : 2013,
-  "yearOfCentury" : 13,
-  "yearOfEra" : 2013
-}
\ No newline at end of file
+"2013-02-13"
```

Comment:
Hvorfor blir diffen så stor når vi bytter? Parset den forrige mapperen altfor mye data?

Reply:
Tidligere versjon av jackson brukte ikke `JodaTimeModule`, så da defaulter den tilbake til å bruke reflection for å serialisere objektet. Og vi fikk da alle de interne feltene til `LocalDateTime` i snapshot-testen våres.

Nyere versjoner godtar ikke at man prøver å serialisere objektene om man ikke har støtte for det, så da måtte vi legge til `JodaTimeModule`. Men den forstår seg mye bedre på hva `LocalDateTIme` er, og velger derfor å bare serialisere det som en enkelt string.